### ISSUE TITLE
Train a backbone from scratch via object detection

### ISSUE THREAD

**Issue by joihn** [open-mmlab/mmdetection#6319]
Do you think it's possible to train a resnet backbone from scratch via object detection ? (aka training the full backbone+neck+head together)
or the gradient would vanish at the end of the head stage ? (I think that's I'm currently experiencing when training from scratch a detectoRS with resnet backbone, but maybe I just need more epoch)

**Comment by ZXW2001** [comment]
hi @joihn i wonder how to train a faster rcnn from scratch, i found 
```
 backbone=dict(
        type='ResNet',
        depth=50,
        num_stages=4,
        out_indices=(0, 1, 2, 3),
        frozen_stages=1,
        norm_cfg=dict(type='BN', requires_grad=True),
        norm_eval=True,
        style='pytorch',
        init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet50')),
```
the above code in  configs/_base_/models/faster-rcnn_r50_fpn.py  , is this mean the code will use the pre-trained weights instead of training from scratch? if i want to training from scratch , what should i do? 
thank you very much!
